CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+Unreleased
+==========
+
+* Implement option to get notifications also for messages not just for mentions
```

Comment:
Don't really think we do this. We add the full changelog upon release of a new version.